jablib/src/main/java/org/jabref/logic/ai/AiExporter.java

```diff
@@ -0,0 +1,106 @@
+package org.jabref.logic.ai;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.jabref.model.entry.BibEntry;
+import org.jabref.model.entry.field.Field;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import dev.langchain4j.data.message.AiMessage;
+import dev.langchain4j.data.message.ChatMessage;
+import dev.langchain4j.data.message.UserMessage;
+
+public class AiExporter {
+
+    private final BibEntry entry;
+
+    public AiExporter(BibEntry entry) {
+        this.entry = entry;
+    }
+
+    public String buildMarkdownExport(String contentTitle, String contentBody) {
+        StringBuilder sb = new StringBuilder();
+        sb.append("## Bibtex\n\n```bibtex\n");
+        sb.append("@").append(entry.getType().getName()).append("{").append(entry.getCitationKey().orElse("")).append(",\n");
+        for (Field field : entry.getFields()) {
+            String value = entry.getField(field).orElse("");
+            sb.append("  ").append(field.getName()).append(" = {").append(value).append("},\n");
+        }
+        sb.append("}\n```\n\n");
+        sb.append("## ").append(contentTitle).append("\n\n");
+        sb.append(contentBody);
```

Comment:
For BibEntry export use  `getStringRepresentation` will be available with https://github.com/JabRef/jabref/pull/14504